go.mod

```diff
@@ -2,17 +2,19 @@ module github.com/siderolabs/discovery-service
 
 go 1.19
 
+replace github.com/siderolabs/discovery-client => github.com/smira/discovery-client v0.0.0-20220915115237-230f317a8e6e
```

Comment:
i guess we can remove this now?

Reply:
once the other PR is merged, I will do that now